src/cms/views/pages/page_actions.py

```diff
@@ -112,6 +116,86 @@ def delete_page(request, page_id, region_slug, language_code):
     )
 
 
+# pylint: disable=unused-argument
+def link_callback(uri, rel):
+    """
+    according to xhtml2pdf documentation this function is neccessary for
+    resolving the django static files references and returning absolute paths
+    to the files
+
+    :param uri: uri that is generated by django template tag 'static'
+    :type uri: string
```

Comment:
```suggestion
    :type uri: str
```
Sphinx is a bit picky when it comes to types :slightly_smiling_face: